src/coord/geo/GeoModel.js

```diff
@@ -148,7 +148,7 @@ var GeoModel = ComponentModel.extend({
      */
     getFormattedLabel: function (name, status) {
         var regionModel = this.getRegionModel(name);
-        var formatter = regionModel.get('label.' + status + '.formatter');
+        var formatter = regionModel.get(['label', status === 'normal' ? '' : status, 'formatter'].join('.'));
```

Comment:
I think this fix is almost correct.
But there is only a problem that 
if the `status` is 'normal',
`['label', status === 'normal' ? '' : status, 'formatter'].join('.')` will get `'label..formatter'`.

Reply:
fixed and I also fix the formatter params & str replacement, please review it